INSTALL.md

```diff
@@ -46,9 +46,10 @@ scst-dkms package instead of the scst package.
 For example, if you want to have dkms support for your SCST rpm install, then
 you would use the following command to make your SCST packages:
 
-    make scst-dkms-rpm
+    make rpm-dkms or make scst-dkms-rpm
```

Comment:
Consider changing this into the following:
```
    make rpm-dkms

or

    make scst-dkms-rpm
```
since indentation with four spaces in markdown documents causes the "computer text" formatting to be used.